suite-native/receive/src/components/ConfirmOnTrezorImage.tsx

```diff
@@ -0,0 +1,37 @@
+import { Image, Pressable } from 'react-native';
+import Animated, { FadeIn } from 'react-native-reanimated';
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { useToast } from '@suite-native/toasts';
+
+const imageContainerStyle = prepareNativeStyle(utils => ({
+    position: 'absolute',
+    bottom: -utils.spacings.medium, // Hides a part of the image under bottom screen edge.
```

Comment:
Why is this necessary? Isn't it just a hack to avoid Screen.tsx spacings and paddings?

Reply:
This is needed to hide part of the image under the bottom edge of the screen (as the comment says). But I think this image will be removed later anyway. The design is still not ready.

Reply:
For that you can just remove paddingBottom in Screen.tsx, is that not a sufficient solution? If we have design where we need to avoid the bottom padding, we should avoid it, not hack it this way. If we need to use this in other screen, we would have to do the same thing again.